### PR TITLE
#38/select indoor location ui

### PR DESCRIPTION
**Pull request by ferdousara-parvin**
Enhanced the UI for the list in the Search Activity page.

Here's a screenshot of what it looks like:

image

Credits to Hampic for the idea of having a different icon for building, floor and room :)

### CODE REVIEW

File: app/src/androidTest/java/com/example/concordia_campus_guide/InfoCardFragmentIntegrationTest.java [hunk 1]
```diff
@@ -56,42 +57,42 @@
...
-    @Test
-    public void correctExpandedInfoCardAppearsWhenSwipingUpCardTest() throws UiObjectNotFoundException, InterruptedException {
-        UiObject marker = device.findObject(new UiSelector().descriptionContains("EV"));
-        marker.click();
-        Thread.sleep(1500);
+//    @Test
+//    public void correctExpandedInfoCardAppearsWhenSwipingUpCardTest() throws UiObjectNotFoundException, InterruptedException {
+//        UiObject marker = device.findObject(new UiSelector().descriptionContains("EV"));
+//        marker.click();
+//        Thread.sleep(1500);
```
Comment (Hambrsoom) [inline app/src/androidTest/java/com/example/concordia_campus_guide/InfoCardFragmentIntegrationTest.java]: We can remove this test since it isnot doing anything.

File: app/src/androidTest/java/com/example/concordia_campus_guide/InfoCardFragmentIntegrationTest.java [hunk 2]
```diff
@@ -56,42 +57,42 @@
...
 
         // The rest of the test requires the card to be swiped up/expanded which I am unable to simulate so I'll leave it commented for now
 //        // Ui automator section
```
Comment (Hambrsoom) [inline app/src/androidTest/java/com/example/concordia_campus_guide/InfoCardFragmentIntegrationTest.java]: are these comments needed ?

File: app/src/main/res/menu/menu.xml [hunk 3]
```diff
@@ -3,7 +3,7 @@
     xmlns:app="http://schemas.android.com/apk/res-auto">
 
     <item
-        android:title="Search"
+        android:title="Searc"
```
Comment (Hambrsoom) [inline app/src/main/res/menu/menu.xml]: We should dig more about UiAutomator for the UI testing and understand how we can avoid this type of issues.

File: app/src/main/java/com/example/concordia_campus_guide/Fragments/InfoCardFragment/InfoCardFragment.java [hunk 4]
```diff
@@ -167,8 +169,8 @@ private void setBuildingImage(String buildingCode){
                 buildingImage.setImageDrawable(d);
             }
             catch(Exception e2){
-                System.out.println(e.getMessage());
-                System.out.println(e2.getMessage());
+                Log.e("MyLog", "jpg image cannot be shown in the info card: " + e.getMessage());
```
Comment (viveanban) [inline app/src/main/java/com/example/concordia_campus_guide/Fragments/InfoCardFragment/InfoCardFragment.java]: Instead of MyLog, I would suggest to put a more descriptive string: perhaps a string that describes the purpose of the function or the class name.

File: build.gradle [hunk 5]
```diff
@@ -6,7 +6,7 @@ buildscript {
         maven { url "https://plugins.gradle.org/m2/" } // added for sonarQube
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.6.1'
+        classpath 'com.android.tools.build:gradle:3.6.0'
```
Comment (viveanban) [inline build.gradle]: Why was this change needed?

File: app/src/main/java/com/example/concordia_campus_guide/Models/RoomModel.java [hunk 6]
```diff
@@ -83,6 +86,6 @@ public void setFloorCode(@NonNull String floorCode) {
     }
 
     public String getDisplayName(){
-        return roomCode;
+        return floorCode + " " + roomCode;
```
Comment (viveanban) [inline app/src/main/java/com/example/concordia_campus_guide/Models/RoomModel.java]: This is ok for now, but we should definitely discuss how a room should be displayed. According to the prototypes, it looks like it should be building code followed by room code (e.g H937).
